Marlin/pins_VOXEL8_GEN3C2.h

```diff
@@ -0,0 +1,161 @@
+#define Z_MAX_PIN               30  // Z max limit switch
+#define Z_ENABLE_PIN            27  // Enable Z stepper driver
+#define Z_MS1_PIN               68  // Z stepper driver microstep select 1
+#define Z_MS2_PIN               67  // Z stepper driver microstep select 2
+
+/************************
+DIGITAL POTENTIOMETER PINS
+*************************/
+// Currently used for limiting motor current
+#undef DIGIPOTSS_PIN
+#define DIGIPOTSS_PIN           38  // Chip Select for digipot SPI communication
+#define DIGIPOT_CHANNELS        {4,5,3,0,1}  // X Y Z E0 E1 digipot channels to stepper driver mapping
+
+/*************************
+    HEATED BED PINS
+*************************/
+#define BED_AVAIL_PIN           82  // From key circuit
+#define HEATER_BED_PIN          4   // Drive the heated bed
+#define TEMP_BED_PIN            2   // Heated bed thermistor
+
+/*************************
+    CARTRIDGE PINS
+*************************/
+#define CART0_SIG0_PIN          5   // PWM Capable
+#define CART0_SIG1_PIN          46  // PWM Capable
+#define CART0_SIG2_PIN          74
+#define CART1_SIG0_PIN          45  // PWM Capable
+#define CART1_SIG1_PIN          75  // Interrupt Capable
+#define CART1_SIG2_PIN          85
+
+/*************************
+	     FFF PINS
+*************************/
+
+#define HEATER_0_PIN            CART0_SIG0_PIN
+#define HEATER_1_PIN            -1 // Set to CART1_SIG0_PIN to use FFF in CART1
+
+/*************************
+  TEMPERATURE SENSE PINS
+*************************/
+#define TEMP_0_PIN              6   // A6 Input Cart0 Therm
+#define TEMP_1_PIN              1   // A1 Input Cart1 Therm
+#define TEMP_2_PIN              -1
+
+/*************************
+    PNEUMATIC PINS
+*************************/
+#if ENABLED(PNEUMATICS)
+  #define PNEUMATIC_PUMP_PIN    7   // Pump driver
+  #define PNEUMATIC_PIN         5   // A5 Input analog tank pressure reading
+  #define REGULATOR_PIN         3   // A3 Input analog output pressure reading
+  #define PNEUMATIC_CART1_PIN   4   // A4 Cartridge 2 analog pressure reading
+#endif
+
+/*************************
+    SOLENOID PINS
+*************************/
+#if ENABLED(PNEUMATICS)
+  #define SOL0_PIN                -1 // Set to CART0_SIG0_PIN for pneumatics in CART0
+  #define SOL1_PIN                CART1_SIG0_PIN
+#endif
+
+
+#define E0_STEP_PIN             34
+#define E0_DIR_PIN              43
+#define E0_ENABLE_PIN           26
+#define E0_MS1_PIN              65
+#define E0_MS2_PIN              66
+
+#define E1_STEP_PIN             33
+#define E1_DIR_PIN              42
+#define E1_ENABLE_PIN           25
+#define E1_MS1_PIN              63
+#define E1_MS2_PIN              64
+
+/**********************************************************
+  Fan Pins
+  Fan_0 8
+  Fan_1 6
+  Fan_2 2
+***********************************************************/
+#define LED_PIN                 13
+#define FAN_PIN                 CART0_SIG2_PIN  // FFF fan toggle
```

Comment:
`CART0_SIG2_PIN` is used for cartridge detection now. If we don't deal with this now, we should submit an issue so we don't forget.